### ISSUE TITLE
Build error on 64-bit Ubuntu 14.04 with gcc 4.8.4 : <stdatomic.h> not found

### ISSUE THREAD

**Issue by mani1996** [cmus/cmus#604]
---
I'm running 64-bit Ubuntu 14.04 LTS with gcc 4.8.4
Running **make install** causes compilation error on Ubuntu with **stdatomic.h** missing

1) `uname -a`:
```
Linux manikantan-Lenovo-G500 3.13.0-95-generic #142-Ubuntu SMP Fri Aug 12 17:00:09 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by flyingmutant** [comment]
Something is probably wrong with your setup -- Travis CI uses 14.04, and everything builds find: https://travis-ci.org/cmus/cmus/jobs/189426569

**Comment by mani1996** [comment]
I believe this was caused by a problem with gcc 4.8 which was shipped with Ubuntu 14.04. The stdatomic.h header was included only from gcc 4.9 onwards. 

Relevant Links:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=58016
https://gcc.gnu.org/gcc-4.9/changes.html

Should I just upgrade to gcc 4.9 and move on?

**Comment by mahkoh** [comment]
GCC versions < 4.9 are no longer supported.
